### DISCUSSION TITLE
Parallax in flex Container?

### DISCUSSION THREAD

**Discussion by visuello** [elementor/elementor#28297] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I want a parallax effect on a flex container, with fixed dimensions like 1600x660px. The problem is, that the image scale to fullscreen. i added 1600x600 as dimensions on a flex container but it didnt help. 

### Describe the solution you'd like.

i hope there is some code for "own css", i cant help myself. sorry. 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by hitsgee** [comment]
Elementor has a dedicated blog post about this with lots of easy to understand examples and CSS for you to use.

https://elementor.com/blog/parallax-effects
